sumOfOther.js

```diff
@@ -0,0 +1,10 @@
+function sumOfOther(arr) {
+  let newArr = [];
```

Comment:
@GoldenKit 'newArr' is never reassigned. Use 'const' instead.
Probably it will be better to rename this variable.